pkg/types/ccipocr3/interfaces.go

```diff
@@ -53,3 +53,9 @@ type ExecutePluginCodec interface {
 type MessageHasher interface {
 	Hash(context.Context, CCIPMsg) (Bytes32, error)
 }
+
+type HomeChainPoller interface {
+	GetConfig() HomeChainConfig
+	Start(ctx context.Context) error
+	Close() error
```

Comment:
Probably better to inherit the `services.ServiceCtx` interface instead of doing this to make it clear what we want to implement

Reply:
I tried but it's a cyclic dependency unfortunately :/

Reply:
Why is it cyclic? That is part of chainlink-common

Reply:
You can just use `services.Service` I think, its in chainlink-common.